Controllers/DeleteTodoController.cs

```diff
@@ -24,7 +25,7 @@ public DeleteTodoController(ITodoService todoService, IMapper mapper)
     }
 
     [HttpDelete]
-    [Authorize("Aserto")]
+    [Aserto]
```

Comment:
Is it possible to use this with a named configuration? If an application needs to use two asserto middleware (with different policies, for example) could it do that (e.g. `[Aserto("config1")]`)?